src/Setup/DevDivVsix/PortableFacades/PortableFacades.swr

```diff
@@ -1,29 +1,30 @@
 use vs
 
 package name=PortableFacades
-        version=1.2.0.0
+        version=1.3.0.0
```

Comment:
It seems like a lot of the binaries changed locations though based on the path changes in the file.  Won't that affect internal references?  

I'm making a lot of assumptions about how this file is used.  Could be very wrong :(